package.json

```diff
@@ -1,11 +1,15 @@
 {
-  "name": "forsetti-ah-frontend",
+  "name": "authors-haven",
   "version": "1.0.0",
-  "description": "Authors Haven - A Social platform for the creative at heart",
+  "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "NODE_ENV=development webpack-dev-server --config ./webpack.config.js --mode development --open",
```

Comment:
Good work @BukkyOmo. I am not sure whether setting the Node environment using `NODE_ENV=development` might work in all environments. We could use the `cross-env` package. What do you think?

Reply:
This has been implemented.